client/src/components/AlumniDirectory.js

```diff
@@ -49,18 +50,36 @@ props:
 - timezone: Number
 */
 export default class AlumniDirectory extends Component {
-    state = {
-        activePage: 1,
-        numEntries: 0,
-        isLoading: false,
-        value: '',
-        totalPages: 0,
-        entries:[],
-        gradYears: [],
-        allText: [],
-        display: [],
-        numResults: 0,
-        filter: 'all'
+    constructor(props) {
+        super(props)
+        this.state = {
+            activePage: 1,
+            numEntries: 0,
+            isLoading: false,
+            value: '',
+            totalPages: 0,
+            entries:[],
+            gradYears: [],
+            allText: [],
+            display: [],
+            numResults: 0,
+            filter: 'all',
+            requestModalOpen: false,
+            alumniDetails: null,
+        }
+        this.openRequestModal = this.openRequestModal.bind(this)
+        this.closeRequestModal = this.closeRequestModal.bind(this)
+    }
+
+    closeRequestModal() {
```

Comment:
nit: can be changed to use one handler `toggleRequestModalOpen` that uses `setState({requestModalOpen: !this.state.requestModalOpen})`